src/core/reducers/errors.js

```diff
@@ -38,16 +38,10 @@ function getMessagesFromError(error) {
         // More info:
         // http://addons-server.readthedocs.io/en/latest/topics/api/overview.html#bad-requests
         value.forEach((msg) => {
```

Comment:
I know this isn't related to your patch but can we expand this variable name? `message` would be nicer and we have the disk space for those extra bytes 😉